dataverse-webapp/src/main/webapp/resources/iqbs/messages.xhtml

```diff
@@ -4,74 +4,72 @@
-        <ui:fragment rendered="#{not empty flash['infoMsg']}">
-            <div class="alert alert-info">
-                <span class="glyphicon glyphicon-info-sign"></span>&#160;<strong>#{bundle['messages.info']}</strong> – <h:outputText escape="false" value="#{flash['infoMsg']} "/>
-            </div>
-        </ui:fragment>
-        <ui:fragment rendered="#{not empty flash['warningMsg']}">
-            <div class="alert alert-warning">
-                <span class="glyphicon glyphicon-exclamation-sign"></span>&#160;<strong>#{bundle['messages.info']}</strong> – <h:outputText escape="false" value="#{flash['warningMsg']} "/>
-            </div>
-        </ui:fragment>
-        <ui:fragment rendered="#{facesContext.validationFailed}">
-            <div class="alert alert-danger">
-                <span class="glyphicon glyphicon-warning-sign"></span>&#160;<strong>#{bundle['messages.validation']}</strong> – #{bundle['messages.validation.msg']}
-            </div>
-        </ui:fragment>
-        <ui:repeat rendered="#{!facesContext.validationFailed}" value="#{facesContext.messageList}" var="msg">
-            #{level = fn:split(msg.severity,' ')[0]; "" }
-            #{bsClass = (level eq 'INFO' ? 'info' : (level eq 'WARN' ? 'warning' : 'danger')); ""}
-            #{bsIcon = (level eq 'INFO' ? 'info' : (level eq 'WARN' ? 'exclamation' : 'warning')); ""}
-            <div class="alert #{cc.attrs.collapsible ? 'alert-dismissable' :''} alert-#{bsClass}">
-                <ui:fragment rendered="#{cc.attrs.collapsible}">
-                    <button type="button" class="close" data-dismiss="alert" aria-hidden="true">×</button>
-                </ui:fragment>
-                <span class="glyphicon glyphicon-#{bsIcon}-sign"></span>&#160;<strong>#{msg.summary}</strong> – <h:outputText escape="false" value="#{msg.detail}" rendered="#{!empty msg.detail and (msg.detail != msg.summary)}"/>
-                <ui:fragment rendered="#{level eq 'FATAL'}">
-                    Please contact <a href="#" onclick="event.preventDefault();PF('contactForm').show();"
-                                      title="Contact #{installationConfig.supportTeamName}">#{installationConfig.supportTeamName}</a> for assistance.
-                    <strong>Date/Time: </strong>
-                    <h:outputText value="#{now}">
-                        <f:convertDateTime pattern="yyyy-MM-dd HH:mm:ss" type="date" />
-                    </h:outputText>
-                </ui:fragment>
-            </div>
-            #{msg.rendered()}
-        </ui:repeat>
-    </cc:implementation>
+<cc:interface>
+    <cc:attribute name="collapsible" default="false"/>
+</cc:interface>
+
+<cc:implementation>
+
+    <!--@elvariable id="globalMessage" type="javax.faces.application.FacesMessage"-->
+    <o:importConstants type="javax.faces.application.FacesMessage"/>
+
+    <p:outputPanel autoUpdate="true">
+        <o:messages globalOnly="true" closable="true" var="globalMessage">
```

Comment:
Seems like `closable="true"` doesn't work. Please check it.

Reply:
Ah yeah it makes sense, since we use our own formatting this property won't apply. The question is, do we want it and write our own closable feature or we don't care about it?

Reply:
To be discussed on weekly I guess. Maybe it's required by WCAG or something

Reply:
As discussed, it is not needed.